packages/cli/src/models/dependency/Dependency.ts

```diff
@@ -43,6 +43,20 @@ export default class Dependency {
     }
   }
 
+  public static hasDependenciesForDeploy(network: string): boolean {
+    const dependencies = ZosPackageFile.getLinkedDependencies() || [];
+    const networkDependencies = ZosNetworkFile.getDependencies(`zos.${network}.json`) || {};
+    const hasDependenciesForDeploy = dependencies.find(depNameAndVersion => {
```

Comment:
Can we use `isDeployedOnNetwork`, or a modified version if needed, to avoid remplementing this logic here?

Reply:
I'll address it in my TODO list, an evaluate in which cases we should stick to class methods and when it's appropriate to instantiate the class.